### PR TITLE
fix: decrease max timeout

### PR DESCRIPTION
**Pull request by q0rban**
The max timeout for screenshots has decreased to 60s. Updating documentation to reflect that.

### CODE REVIEW

**Comment by q0rban** [comment]
@rabbitlair is this correct? Or should it be 120?

**Comment by rabbitlair** [comment]
We increased it to 120 seconds from 60 seconds, yes.

File: content/reference/tugboat-configuration.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ Tugboat default values, and can also be overridden per-URL.
 | :---------- | :------ | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `enabled`   | Boolean | `true`  | Whether to render Screenshots for the URLs defined for this Service                                                                                                              |
 | `fullPage`  | Boolean | `true`  | Whether to use the default `fullPage` method. Disabling this uses an alternative that is more friendly to elements that have `vh` CSS Styles, but can sometimes be less accurate |
-| `timeout`   | Number  | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `300`                                                                       |
+| `timeout`   | Number  | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `60`                                                                        |
```
Comment (q0rban) [inline content/reference/tugboat-configuration.md:320]:
```suggestion
| `timeout`   | Number  | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `120`                                                                        |
```

File: content/reference/tugboat-configuration.md [hunk 2]
```diff
@@ -350,7 +350,7 @@ screenshots taken of the Base Preview used to compare to screenshots taken of th
 | :---------- | :--------------- | :------ | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `enabled`   | Boolean          | `true`  | Whether to render Visual Diffs for the URLs defined for this Service. Visual Diffs depend on Screenshots being enabled. If Screenshots are disabled, this setting has no effect.     |
 | `fullPage`  | Boolean          | `true`  | Whether to use the default `fullPage` method. Disabling this uses an alternative that is more friendly to elements that have `vh` CSS Styles, but can sometimes be less accurate     |
-| `timeout`   | Number           | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `300`                                                                           |
+| `timeout`   | Number           | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `60`                                                                            |
```
Comment (q0rban) [inline content/reference/tugboat-configuration.md:353]:
```suggestion
| `timeout`   | Number           | `30`    | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `120`                                                                            |
```

File: content/reference/tugboat-configuration.md [hunk 3]
```diff
@@ -507,7 +507,7 @@ either a string, such as `/blog`, or a map overriding the following screenshot o
 | :-------- | :--------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | url       | _none_     | The relative URL to create the visual diff for. This option is required                                                                                                             |
 | aliases   | `:default` | Only create visual diffs for these [Service aliases](#aliases). The special `:default` alias can be used to target the automatically generated Service URL (the one with no alias). |
-| timeout   | `30`       | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `300`                                                                          |
+| timeout   | `30`       | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `60`                                                                           |
```
Comment (q0rban) [inline content/reference/tugboat-configuration.md:510]:
```suggestion
| timeout   | `30`       | How long to wait for a page to be ready when taking a screenshot, in seconds. Minimum: `1`, Maximum: `120`                                                                           |
```

**Pull request merged**